crates/holochain/src/core/workflow/sys_validation_workflow.rs

```diff
@@ -1,4 +1,77 @@
-//! The workflow and queue consumer for sys validation
+//! ### The sys validation workflow
+//!
+//! This workflow runs against all [`Action`]s that are going into the DHT database. Either coming from the authored database or from other nodes on the network.
+//!
+//! The purpose of the workflow is to make fundamental checks on the integrity of the data being put into the DHT. This ensures that invalid data is not served
+//! to other nodes on the network. It also saves hApp developers from having to write these checks themselves since they set the minimum standards that all data
+//! should meet regardless of the requirements of a given hApp.
+//!
+//! #### Validation checks
+//!
+//! The workflow operates on [`DhtOp`]s which combine [`Action`]s give context to [`Action`]s and carry [`Entry`]s where relevant. Checks that you can rely on
+//! sys validation having performed are:
+//! - For a [`DhtOp::StoreRecord`]
+//!    - Run the [store record checks](#store-record-checks).
+//!    - If the [`Entry`] is an [`Entry::CounterSign`] then the countersigning session data is mapped to a set of [`Action`]s and each of those actions must be be found locally before this op can progress.
+//!    - The [`Action`] must be either a [`Action::Create`] or an [`Action::Update`].
```

Comment:
I read this as "a create action", because the enum variant is the actual subject here. But I agree it reads weirdly if you don't happen to do that. Not sure what's best here

Reply:
Well in many places you do "an" like in this same sentence `an [Action::Update]`. I stumble over a [Action...]. Whichever way, I'd use it consistently.

Reply:
Looking into what [dictionaries](https://www.merriam-webster.com/grammar/is-it-a-or-an) and style guides say, and it's all about the pronunciation. And there are even cases where people pronounce words differently and use a different article ("a historical" / "an historical").

So, @ThetaSinner would pronounce `Action::Create` as "create", and @jost-s (and myself) would pronounce it "action create". Both seem valid. The question is if we want to standardize on one or the other. 

There is at least one other thing we don't standardize on, which is which variety of English we're using. Sometimes we spell it "neighborhood", sometimes "neighbourhood".

We could decide on this, but I'm actually more happy to have these inconsistencies around as little reminders of Holochain's pluralistic values. I would not stand for "a `Action::Update`" though :)